src/DynamicMvvm.Reactive/Command/IDynamicCommandExtensions.cs

```diff
@@ -6,7 +6,7 @@
 namespace Chinook.DynamicMvvm
 {
 	/// <summary>
-	/// Extensions on <see cref="IDynamicCommand"/>.
+	/// Extensions on <see cref="IDynamicCommand"/> to observe its executing state.
```

Comment:
That's probably too specific. I would keep the original version. Same for the next file.